src/Microsoft.IdentityModel.Tokens/ECDsaSecurityKey.cs

```diff
@@ -90,6 +108,10 @@ public override SignatureProvider GetSignatureProvider(string algorithm, bool ve
                 return CryptoProviderFactory.CreateForSigning(this, algorithm);
         }
 
+        /// <summary>
+        /// true if this <see cref="ECDsaSecurityKey"/> supports the algorithm; otherwise, false.
```

Comment:
"Returns whether the ECDsa member supports the given algorithm." And then there should be a separate "returns" section.